tests/lax_numpy_operators_test.py

```diff
@@ -731,6 +731,24 @@ def testSpacingSubnormals(self, dtype):
     self._CheckAgainstNumpy(np.spacing, jnp.spacing, args_maker, check_dtypes=True, tol=0)
     self._CompileAndCheck(jnp.spacing, args_maker, tol=0)
 
+  @jtu.ignore_warning(category=RuntimeWarning, message="divide by zero.*")
+  @jtu.ignore_warning(category=RuntimeWarning, message="invalid value.*")
+  def testFloorDivideByZeroInt(self):
+    x1 = jnp.eye(3, dtype=int)
+    x2 = jnp.zeros((3, 3), dtype=int)
+    jnp_result = jnp.floor_divide(x1, x2)
+    np_result = np.floor_divide(x1, x2)
+    self.assertAllClose(jnp_result, np_result, check_dtypes=True)
+
+  @jtu.ignore_warning(category=RuntimeWarning, message="divide by zero.*")
+  @jtu.ignore_warning(category=RuntimeWarning, message="invalid value.*")
+  def testFloorDivideFloatSpecialCases(self):
+    x1 = jnp.array([-np.nan, -np.inf, -2., -1., -0., 0., 1., 2., np.inf, np.nan])
+    x2 = 0.
```

Comment:
Why not test all the special values in `x2` as well?

Reply:
Thanks! Test case is modified so as to test for all the specific values in `x2`.